contracts/src/governance/README.md

```diff
@@ -1,6 +1,6 @@
-## Utility Contracts
+# ValidatorSetSig Contract
```

Comment:
```suggestion
# Governance Contracts
```

nit: Can we change this back and `## Overview` below to `## ValidatorSetSig Contract` now that we have a `governance` folder?